my-app/src/pages/BodyCube/BodyUsersList/BodyUsersList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useAppSelector } from 'core/interfaces/interfaces';
+import { nanoid } from 'nanoid';
+import clN from './bodyUsersList.module.scss';
+import BodyUser from './BodyUser/BodyUser';
+
+function BodyUsersList() {
+	const users = useAppSelector(state => state.leaders.usersLeaders[state.leaders.day]);
+	return (
+		<div className={clN.bodyUsersList}>
+			{!!users &&
+				users.map((el, i) => {
+					return <BodyUser key={nanoid()} name={el.name} score={el.score} listNumber={i} photo={el.photo} />;
```

Comment:
`key={nanoid()}` is a bad practice. Read about it in the article below (in the section about Nanoid)

https://sudonull.com/post/13044-4-libraries-that-simplify-the-life-of-a-React-developer

Reply:
chenged for el.id  ;
el.id = nanoid() in getRequest